Midstone/ComponentFramework/Scene5g.cpp

```diff
@@ -230,12 +252,16 @@ void Scene5g::Update(const float deltaTime) {
 		frameCounter++;
 	}
 	shipPos.x -= deltaTime / 10;
-	shipModelMatrix = MMath::translate(shipPos) * MMath::rotate(90.0f, Vec3(0.0f, 1.0f, 0.0f));
+	shipModelMatrix = MMath::translate(shipPos * 40) * MMath::rotate(90.0f, Vec3(0.0f, 1.0f, 0.0f));
```

Comment:
What does 40 means?

Reply:
Thats my test scene I just moved the test ship out of the way to test the explosion